stories/topics/proc-gcp-run-upgrade-playbook

```diff
@@ -0,0 +1,20 @@
+[id="proc-gcp-run-upgrade-playbook"]
+
+= Running the upgrade playbook
+
+The following instructions describe how to run the upgrade playbook.
+
+.Procedure
+. Pull the docker image for the `ansible-on-clouds-ops` container with a tag the same as the version of the foundation deployment.
++
+[NOTE]
+=====
+The `ansible-on-clouds-ops` container image tag must match the version of your foundation deployment. 
+For example, if your foundation deployment version is 2.2.1, pull the `ansible-on-clouds-ops` image with tag 2.2.1.
+=====
++
+[source,bash]
+----
+$ export IMAGE=quay.io/ansible-on-clouds/ansible-on-clouds-ops:2.2.1
+$ docker pull $IMAGE
+----
```

Comment:
In every case so far there's been a different file for EMEA, something like quay.io/ansible-on-clouds/ansible-on-clouds-ops-dev:2.3-latest-rh_limited